### ISSUE TITLE
Transition ufs-weather-model to WCOSS2

### ISSUE THREAD

**Issue by BenjaminBlake-NOAA** [ufs-community/ufs-weather-model#1116]
The ufs-weather-model code will need to be moved to WCOSS2 in order for real-time experimental RRFS runs to continue.  WCOSS1 is scheduled to go offline on July 15th.

**Comment by junwang-noaa** [comment]
A quick update, we are working with nceplibs team to get libraries ready on wcoss2 (hps-stack issue [#432](https://github.com/NOAA-EMC/hpc-stack/issues/432)). Some libraries were updated on wcoss2 according to nceplibs team, those libraries need to be installed on all ufs supported platforms too.

**Comment by DusanJovic-NOAA** [comment]
'wcoss2' branch in my fork can now be used on wcoss2 (cactus). Only -DAPP=ATM and ATMW for now due to missing pio, and 3 other NASA libraries.

`git clone --recursive --branch wcoss2 https://github.com/DusanJovic-NOAA/ufs-weather-model`

**Comment by BenjaminBlake-NOAA** [comment]
@DusanJovic-NOAA  I was able to compile the ufs-weather-model code on Cactus (with -DAPP=ATM) using your branch, thanks for getting that set up.  One issue I ran into initially was that the cray-mpich/8.1.7 module was not being loaded, even though it is in modulefiles/ufs_wcoss2.  After adding module load envvar/1.0 to the top of that modulefile, the cray-mpich/8.1.7 module was successfully loaded.
